requirements-torch.txt

```diff
@@ -0,0 +1,4 @@
+torch~=1.4.0
```

Comment:
Is there a specific reason to keep using 1.4.0? Latest is 1.5.0.

Reply:
updated to 1.5.0